radio_beam/beam.py

```diff
@@ -631,11 +633,11 @@ class EllipticalTophat2DKernel(Kernel2D):
 
     _is_bool = True
 
-    def __init__(self, width, height, position_angle, support_scaling=8, **kwargs):
+    def __init__(self, width, height, position_angle, support_scaling=2, **kwargs):
 
         self._model = EllipticalDisk2D(1. / (np.pi * width * height), 0, 0,
                                        width, height, position_angle)
         self._default_size = _round_up_to_odd_integer(support_scaling *
-                                                      np.max([width, height]))
+                                                      np.sqrt(height**2 + width**2))
```

Comment:
Because it's elliptical, I still think we want the max of the 2 axes, not their hypotenuse.   The max should accommodate any shape.  Though since you also changed the `support_scaling` default for the disk... could you explain your reasoning?